### PR TITLE
Refactor to fix Edge support

### PR DESCRIPTION
**Pull request by rsinger**
In the run-up for Chrome and Firefox releases, Edge functionality was broken.

This reconciles the differences between the two and removes all of the Edge-specific code (minus the grunt task required for Edge's manifest.json).

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -63,9 +63,17 @@ module.exports = function(grunt) {
...
       manifest.applications = { 'gecko': { 'id': grunt.option('identifier') } };
     }
-    if (browser !== 'msedge') {
-      delete(manifest['-ms-preload']);
-    }
+    if (browser === 'msedge') {
+      manifest.icons = [
+        {"sizes":"16x16", "src": "images/icon16.png"},
+        {"sizes": "30x30", "src": "images/icon30.png"},
+        {"sizes": "50x50", "src": "images/icon50.png"},
```
Comment (rgubby) [inline Gruntfile.js]: I think the docs in the README need updating with these additional image sizes for Edge

File: manifest.json [hunk 2]
```diff
@@ -38,9 +40,5 @@
           }
     },
     "default_locale": "en",
-    "manifest_version": 2,
-    "-ms-preload": {
-        "backgroundScript": "js/ms/backgroundScriptsAPIBridge.js",
-        "contentScript": "js/ms/contentScriptsAPIBridge.js"
-    }
+    "manifest_version": 2
```
Comment (rgubby) [inline manifest.json]: Does this need to be bumped? Does it matter?

File: js/bookmarker.js [hunk 3]
```diff
@@ -4,18 +4,27 @@
  * Edge needs this to be a content_script that is injected, however.
```
Comment (rgubby) [inline js/bookmarker.js]: Is this line true anymore?

File: Gruntfile.js (hunk 1 above)
Comment (rgubby) [reply Gruntfile.js]: Also does the icon set have to be set like this for Edge? Or could you just add this lot to the main set of manifest icons?

Comment (rsinger) [reply Gruntfile.js]: manifoldjs, annoyingly, uses this completely different icons schema.

File: manifest.json (hunk 2 above)
Comment (rsinger) [reply manifest.json]: No, that's the version of the style of manifest.

File: js/bookmarker.js (hunk 3 above)
Comment (rsinger) [reply js/bookmarker.js]: It is, but also for every other browser, with this change.

**Pull request merged**